.github/workflows/nightly.yml

```diff
@@ -0,0 +1,37 @@
+---
+# A schedule only runs on branches that match include rules from _both_ `main`
+# and the branch itself.  On `main`, we blanket include all branches that might
+# want to be enabled, then particular branches can override it to exclude
+# themselves by removing themselves from the trigger list.  For example, old
+# stable branches can remove `stable/*` from their copy of this file once they
+# reach their end-of-life.
+name: Nightly Main tests
+on:
+  schedule:
+    - cron: "20 6 * * *"
+  workflow_dispatch:
+
+jobs:
+  nightly-tests:
+    # Nightly cron job.
+    #
+    # For this to run on a branch, the `schedules` trigger  up at the top of this
+    # file needs to match on _both_ `main` and the branch itself.
+    if: ${{ github.repository_owner == 'Qiskit'}}
+    uses: ./.github/workflows/on-nightly.yml
```

Comment:
Why does this need to be a separate file. I can understand on the other triggers where there is a clear divide between multiple job types. But for this it just seems like unnecessary layer because this is already defined in using the split job definitions. I would flatten this so there is only one nightly file to limit how much jumping around you need to do to understand the configs.

Reply:
Removed "nightly.yml" in favor of "on-nightly" in 4655adc, also updated that comment.